uktrade/data-hub-frontend
# Company postcode filter

## Description of change

Adds company postcode filter - note that this also allows you to enter multiple postcodes (comma separated)

I have also added some tests to ensure that the correct params are added when calling the api endpoint

## Test instructions

Go to '/companies/react' and apply the postcode filter - also try with comma separated values. A chip should appear and the url should als be updated accordingly.

## Screenshots

![Screenshot from 2021-06-11 17-37-36](https://user-images.githubusercontent.com/1234577/121721105-c9f72d00-cadb-11eb-831c-e7aca710108f.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
